v_4_9_1/files/k8s-resource/calico-all.yaml

```diff
@@ -7,13 +7,9 @@
 #  - Added resource limits to install-cni.
 #  - Added 'priorityClassName: system-cluster-critical' to calico daemonset.
 #
-# Calico Version v3.9.1
+# Originally intended for Calico version v3.9.1
```

Comment:
This might work for new minor/patch releases of Calico. This just makes it clear that it was originally tested for this version.